components/SQL/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import classNames from "classnames";
+import hljs from "highlight.js/lib/highlight";
+import sql from "highlight.js/lib/languages/sql";
+
+import styles from "./style.module.scss";
+
+type Props = {
+  sql: string;
+  inline?: boolean;
+};
+
+// Keywords that should not be highlighted
+const unhighlightedKeywords = [
+  "oid",
+  "date",
+  "year",
+  "month",
+  "unique",
+  "desc",
+  "asc",
+  "interval",
+  "null",
+  "not",
+  "upsert",
+  "name",
+  "position",
+  "is",
+  "id",
+  "cleanup",
+  "query",
+  "extract",
+  "coalesce",
+];
+
+class SQL extends React.Component<Props> {
```

Comment:
Copying this component in here because it's clearly needed in docs, but it's also needed in-app for non-documentation issues. The right solution is, as we've discussed, a separate shared component package, but that complicates things a bit. I think a copy is okay for now, but open to other ideas.